ui/src/tool-panel/infobox/infobox.module.scss

```diff
@@ -130,7 +130,31 @@
     flex: 1;
 }
 
-.infobox-tabs {
+.tool-panel__tab-bar {
     top: 0;
     background: vayla-design.$color-white-default;
+    position: sticky;
+    z-index: 1;
+    box-shadow: 0 1px 1px rgba(0, 0, 0, 0.12);
+}
+
+.tool-panel__tab {
+    display: inline-block;
+    padding: 6px 8px 6px 8px;
+    border-bottom: 3px solid transparent;
+    cursor: pointer;
+    max-width: 190px;
```

Comment:
TIketillä speksattiin maksimileveydeksi "noin 20 merkin verran." Tähän ei kyllä 20 M-kirjainta mahdu, mutta normaalimpaa tekstiä se on aika lähelle. Samalla se on tasan puolet tabipalkin leveydestä, eli nyt juuri ja juuri mahtuu kaksi maksimilevyistä tabia vierekkäin.